sql/wsrep_mysqld.cc

```diff
@@ -1488,9 +1489,10 @@ static bool wsrep_prepare_keys_for_isolation(THD *, const char *db,
     if (!wsrep_prepare_key_for_isolation(db, table, ka)) goto err;
   }
 
-  for (const TABLE_LIST *table = table_list; table;
-       table = table->next_global) {
-    if (!wsrep_prepare_key_for_isolation(table->db, table->table_name, ka))
+  for (const TABLE_LIST *table_it = table_list; table_it;
```

Comment:
It's a formatting change, so not really a conflict resolution... but I didn't want to create a separate commit for it.
I also want to enable the circleci formatting check in a separate pr, to avoid formatting issues like this in the future.